### DISCUSSION TITLE
Probleme bei der Einrichtung / Setup problems

### DISCUSSION THREAD

**Discussion by ghost** [Kovah/LinkAce#665] [General]
Habe Linkace installiert, das einloggen und ausloggen funktioniert auch, aber das gesamte Layout stimmt überhaupt nicht. Wenn ich die Seite aufrufe https://mk12.site/linkace/public/, habe ich eine seltsame Situation, bezüglich der Gestaltung…
Ich würde auch gerne die Post-Setup Steps vornehmen, in der .env sind aber nur die Zugangsdaten zur Datenbank zu finden. Weitere Einstellung für den E-Mail Versand sind in meiner .env leider nicht zu finden…
Kann mir bitte jemand helfen …

---------------------------------------------------------------

I installed Linkace, logging in and out works, but the overall layout isn't right at all. When I go to the page https://mk12.site/linkace/public/, I have a strange situation regarding the design...
I would also like to do the post-setup steps, but the .env only contains the access data to the database. Unfortunately, there are no other settings for sending e-mails in my .env...
Can someone help me please …

**Comment by Kovah** [comment]
Hi, der Web Server (Apache?) muss zwingend als Document Root das `linkace/public` Verzeichnis gesetzt haben, und nicht das `linkace` Verzeichnis.

Bitte auch beachten, dass die Installation in einem Unterverzeichnis nicht getestet ist und unter Umständen nicht korrekt funktioniert.

**Reply by ghost** [reply]
Dann sollte man das vielleicht auch verständlich in der Installationsanleitung hineinschreiben, und vor allem auch erklären, wie dies bewerkstelligt wird?
Es gibt auch Leute, die haben mit solchen Sachen keine Erfahrung. 
Wie, und das ist meine Frage, kann ich das Problem jetzt lösen. Über eine konkrete Anleitung diesbezüglich würde ich mich für sehr freuen!

**Reply by ghost** [reply]
Jetzt habe ich's geändert, und es wird gar nichts mehr angezeigt, warum kann man nicht einfach Anwendung erstellen, die sich so einfach installieren lassen wie WordPress? Ihr Programmierer geht immer davon ausgehen, dass die Anwender mit der Installation von solchen Systemen Erfahrung haben. Ich würde gerne mit einem solchen System arbeiten als Anwender. Und es nervt, wenn man nicht vorankommt, weil keine vernünftigen Installationsanleitungen vorhanden sind.

**Reply by Kovah** [reply]
Im Fall von LinkAce ist der Grund ganz einfach: ich habe schlichtweg keine Zeit dafür, die Anwendung für _alle_ zugänglich zu machen. Ich habe sie für mich gebaut und bin bereits große Schritte gegangen, damit sie andere überhaupt installieren und nutzen können. Wenn dafür ein gewisses technisches Verständnis nötig ist, dann ist das für mich okay.

Der Vergleich mit WordPress ist an der Stelle auch recht unfair. Hinter der Software steht eine Firma mit bestimmt einem Dutzend Angestellten die dafür Vollzeit bezahlt werden, um sie für jeden Ottonormalverbraucher installierbar zu machen. Mich bezahlt niemand auch nur ansatzweise für meine Arbeit an LinkAce.

Um auf dein Problem zurückzukommen: in [der Anleitung](https://www.linkace.org/docs/v1/setup/setup-without-docker/#4-point-your-web-server-to-public) ist erklärt, dass der Webserver auf public zeigen muss. Jeder Webserver kann das, aber es würde den Rahmen sprengen das für jeden einzeln zu erklären. Dafür gibt es die Dokumentation der Webserver.
Da du keine weiteren Infos über dein Setup angegeben hast kann ich dir auch nicht weiter helfen.
